src/core/internal/modes/relay.ts

```diff
@@ -51,6 +51,11 @@ export function relay(config: relay.Parameters = {}) {
 
   return Mode.from({
     actions: {
+      async addFunds(_parameters) {
+        // TODO: temporarily returning a value so it doesn't throw
+        return '0x'
+      },
```

Comment:
Maybe we amend [this](https://github.com/ithacaxyz/porto/blob/main/src/remote/Actions.ts#L74) to accept `result` if it is provided as an option to bypass `provider.request`?

```suggestion
      async addFunds(_parameters) {
        throw new Provider.UnsupportedMethodError()
      },
```